control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1575,6 +1575,41 @@ paths:
           $ref: '#/components/responses/ServerError'
       security:
         - JWT: []
+  '/volumes{volume_id}/snapshots/{snapshot_name}':
```

Comment:
```suggestion
  '/volumes/{volume_id}/snapshots/{snapshot_name}':
```

Reply:
it's still a valid path I guess, just not the one we want

Reply:
Well, I am not sure if this qualifies as valid path, because with the `/` the url becomes variable right. Anyways that's a different discussion altogether.

Reply:
It's a valid path :) 
/volumes6a666979-c222-4278-9aff-72053835abdc/snapshots/my-snap-1

Reply:
No, what I mean is individually its an valid path but based on the value of `{volume_id}` the whole url would keep changing.